cmake/fckit_preprocess_fypp.cmake

```diff
@@ -192,7 +171,7 @@ function( fckit_target_preprocess_fypp _PAR_TARGET )
 
   set( options NO_LINE_NUMBERING )
   set( single_value_args "" )
-  set( multi_value_args FYPP_ARGS FYPP_ARGS_EXCLUDE DEPENDS )
+  set( multi_value_args FYPP_ARGS FYPP_ARGS_EXCLUDE DEPENDS INCLUDEDIRS )
```

Comment:
To be consistent with `ecbuild_add_executable` we can name this `INCLUDES` and also add `DEFINITIONS`
```
                        [ INCLUDES <path1> [<path2> ...] ]
                        [ DEFINITIONS <definition1> [<definition2> ...] ]
```